app/models/subscription.rb

```diff
@@ -0,0 +1,7 @@
+class Subscription < ApplicationRecord
+  belongs_to :user
+  belongs_to :plan
+
+  validates :user_id, uniqueness: { scope: :plan_id,
+    message: "Can not subscribe to same plan again ! " }
```

Comment:
update it to single line
```suggestion
  validates :user_id, uniqueness: { scope: :plan_id, message: "Can not subscribe to same plan again !" }
```